website/content/cli/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Introduction
+sidebar_position: 0
```

Comment:
I think the sidebar position here gets removed, and then when you click on the category label you get this page. Right now I have to expand and then click on Introduction to get the landing page.